### ISSUE TITLE
Installing ibm-db installs a global "tests" module

### ISSUE THREAD

**Issue by vijaykiran** [ibmdb/python-ibmdb#764]
When installing using `pip install ibm-db` the `tests` module is installed into site-packages. This is breaking a project with `tests` used as namespaced package.

Looks like modules contains both `tests` and `testfunctions` included in the build:
https://github.com/ibmdb/python-ibmdb/blob/937e3f0395e6d0d56e9cefcfe346de1596c9a845/IBM_DB/ibm_db/setup.py#L442 

Is it possible to exclude it from the installs or rename them into proper namespace?

- Operating System Name: macOS
- Python Version: 3.9.12
- ibm_db version: 3.1.2

**Comment by Earammak** [comment]
@vijaykiran , I would suggest you to please try to change your project name only. we cannot change this as it is maintained from day 1.

**Comment by m1n0** [comment]
@Earammak naming an installed package `tests` is not a good Python practice, it produces very confusing issues for anyone using the same name in their projects, which is actually quite common. If the `tests` package is required in the installed library then renaming them is a simple refactor any IDE can do in a few seconds, I fail to see how 

> we cannot change this as it is maintained from day 1.

is relevant to the issue. Badly named package is a badly named package, no matter how long has it been named like that.

I did a quick search in the repo for `tests` and I did not see any direct import of the module, is the `tests` module (and `testfunctions` as well for that matter) even needed for the library to work? It seems both are used in tests only, so no need to distribute them to users right? If they are removed from the installed package then renaming is not needed of course as not installing them would make those two packages invisible to the outside world.

This would be preferred as shipping your test code alongside the library is far from ideal.
